### DISCUSSION TITLE
Include test harnesses for PrimeNg components

### DISCUSSION THREAD

**Discussion by azaeng04** [primefaces/community#901] [PrimeNG] [answered]
Angular introduced component harnesses from version 9 if I am not mistaken.
https://material.angular.io/cdk/test-harnesses/overview

Currently PrimeNg have no harnesses that can be used to interact with the relevant components.
It would be helpful if this were to be included.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've plans to update tests in the future after passthrough and unstyled mode implementations.
